Fix dependency conflict and wrong badge link. (#93)
src-d/modelforge: requirements.txt

scipy==1.0.0
clint==0.5.1
google-cloud-storage==1.2.0

> # Requests should be first to avoid dependency problems on urllib3

Reviewer: ```suggestion
# requests should be before dulwich to avoid urllib3 version conflict
```
